pkg/platform/gitlab.go

```diff
@@ -293,6 +293,10 @@ func (g *GitLab) ReportEntrypointsSummary(ctx context.Context, p *EntrypointsSum
 
 // ClearReports clears any existing reports that can be removed.
 func (g *GitLab) ClearReports(ctx context.Context) error {
+	if err := validateGitLabReporterInputs(g.cfg); err != nil {
```

Comment:
does this need added for any others? e.g. ListReports

Reply:
You're right 🥇